packages/host/app/components/matrix/login.gts

```diff
@@ -171,11 +136,6 @@ export default class Login extends Component<Signature> {
     );
   }
 
-  @action handleSubmit(event: SubmitEvent) {
```

Comment:
Why remove this? https://github.com/cardstack/boxel/pull/885

Reply:
Instead of using form submit event, I updated it to handle `enter` key [041d24f](https://github.com/cardstack/boxel/pull/921/commits/041d24fa171aecd4169d927fa85ed7c549bf0ca7).

Reply:
Why not use submit event? The advantage is that you don't have to attach a handler to every input

Reply:
It seems the 'forgot password?' text button breaks the submit event because 'forgot password?' text button is the first button after the input fields and not the login button. So, when user press 'enter' it will trigger 'forgot password?' text button instead of triggering the login button.